.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us.
+
+    Note you do not need to provide all the fields given but the more you give the better and easier for us to figure
+    out the issue.
+-->
+
+Related: <!-- Any issues or PRs that your PR relates to -->
+
+Use cases:
+* [ ] <!-- Provide atleast one use case -->
```

Comment:
Provide a use case if this PR is a new feature and does not have an associated issue.